sceptre/connection_manager.py

```diff
@@ -92,22 +92,23 @@ class ConnectionManager(object):
     _clients = {}
     _stack_keys = {}
 
-    def __init__(self, region, profile=None, stack_name=None, iam_role=None):
+    def __init__(self, region, profile=None, stack_name=None, iam_role=None, iam_role_session_duration=3600):
         self.logger = logging.getLogger(__name__)
 
         self.region = region
         self.profile = profile
         self.stack_name = stack_name
         self.iam_role = iam_role
+        self.iam_role_session_duration = iam_role_session_duration if iam_role_session_duration else 3600
```

Comment:
When would this value be falsey? I'm not sure when this would be applicable, considering your code.

Also, pro tip: You could do this same thing in a less verbose way like:
```python
        self.iam_role_session_duration = iam_role_session_duration or 3600
```